lib/CustomView/Config.php

```diff
@@ -22,6 +22,16 @@
  */
 final class Config
 {
+    /**
+     * @deprecated Will be removed in Pimcore 11
+     */
+    private const STORAGE_DIR = 'PIMCORE_CONFIG_STORAGE_DIR_CUSTOM_VIEWS';
+
+    /**
+     * @deprecated Will be removed in Pimcore 11
+     */
+    private const WRITE_TARGET = 'PIMCORE_WRITE_TARGET_CUSTOM_VIEWS';
```

Comment:
I think we don't need this constants as we use the value  just once anyway, right?